src/Stack/Package.hs

```diff
@@ -1062,6 +1064,7 @@ getCabalFileName
     => Path Abs Dir -- ^ package directory
     -> m (Path Abs File)
 getCabalFileName pkgDir = do
+    liftIO $ hpack pkgDir
```

Comment:
Ok, makes sense.  I'll change that some time soon (may have to wake until next weekend).